src/accelerate/utils/environment.py

```diff
@@ -176,3 +176,22 @@ def check_fp8_capability():
     """
     cuda_device_capacity = torch.cuda.get_device_capability()
     return cuda_device_capacity >= (8, 9)
+
+
+def get_cpu_distributed_information():
+    """
+    Attempts to return various information about the environment in relation to CPU distributed training
```

Comment:
Why "Attempts"? This made me think that this function may raise an error if it can't find the info.